test/models/user_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  
+  def setup
+    foo = ('a'..'z').to_a.shuffle.first(8).join # random 8char string for unique email
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.